data_dir/scylla.yaml

```diff
@@ -6,8 +6,8 @@ test_duration: 60
 stress_cmd: cassandra-stress write cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..10000000
 # tcpdump to try to capture nodetool API
 tcpdump: false
-# Number of database nodes
-n_db_nodes: 6
+# Number list of database nodes in multiple date centers. Legacy type (int): Number of database nodes
```

Comment:
The parameter is a list string. each iteam is the nodes number of the single dc.

n_db_nodes: '1 2 3'  ===> dc1 has 1 node, dc2 has 2 nodes, dc3 has 3 nodes.

The parameter also indicates the number of dcs.